tests/failover.go

```diff
@@ -106,6 +108,30 @@ func (oa *operatorActions) TruncateSSTFileThenCheckFailover(info *TidbClusterCon
 			}
 			return true, nil
 		})
+	if err != nil {
+		glog.Errorf("failed to check truncate sst file: %v", err)
+		return err
+	}
+
+	if err := wait.Poll(1*time.Minute, 30*time.Minute, func() (bool, error) {
+		if err := tikvOps.RecoverSSTFile(info.Namespace, podName); err != nil {
+			glog.Errorf("failed to recovery sst file %s/%s, %v", info.Namespace, podName, err)
+			return false, nil
+		}
+
+		return true, nil
+	}); err != nil {
+		return err
+	}
+
+	glog.Infof("deleting pod: [%s/%s] again", info.Namespace, store.PodName)
+	return wait.Poll(10*time.Second, time.Minute, func() (bool, error) {
+		err = oa.kubeCli.CoreV1().Pods(info.Namespace).Delete(store.PodName, &metav1.DeleteOptions{})
+		if err != nil {
+			return false, nil
+		}
+		return true, nil
+	})
```

Comment:
check the pod is recreated?

Reply:
yes, we have check it recreated in later logic